documentation/README.md

```diff
@@ -262,11 +260,3 @@ Step-up flow for a single user may include multiple API calls like GetUser, GetU
 ## Conclusion
 
 In this blog post, you saw how to implement step-up or step-up authentication for privileged APIs.  The solution uses Amazon Cognito User Pools for user management.
-## Security
-
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
-
-## License
-
-This library is licensed under the MIT-0 License. See the LICENSE file.
```

Comment:
Any reason for removing the `License` section? (since the LICENSE file is still MIT, we might as well leave it in). Let's maybe add a section at the end pointing to the original source of the sample (to retain AWS' authorship), and mentioning that the original sample was licensed under MIT-0 License. 👍